app/Locale/da_DK/translations.php

```diff
@@ -1,1378 +1,1378 @@
-    // '%d notification' => '',
-    // 'There is no external integration installed.' => '',
-    // 'You are not allowed to update tasks assigned to someone else.' => '',
-    // 'You are not allowed to change the assignee.' => '',
-    // 'Task suppression is not permitted' => '',
-    // 'Changing assignee is not permitted' => '',
-    // 'Update only assigned tasks is permitted' => '',
-    // 'Only for tasks assigned to the current user' => '',
-    // 'My projects' => '',
-    // 'Your are not member of any project.' => '',
-    // 'My subtasks' => '',
-    // '%d subtasks' => '',
-    // '%d subtask' => '',
-    // 'Only moving task between those columns is permitted for tasks assigned to the current user' => '',
-    // '[DUPLICATE]' => '',
-    // 'DKK - Danish Krona' => '',
-    // 'Remove user from group' => '',
-    // 'Assign the task to its creator' => '',
-    // 'This task was sent by email to "%s" with subject "%s".' => '',
-    // 'Predefined Email Subjects' => '',
-    // 'Write one subject by line.' => '',
-    // 'Create another link' => '',
-    // 'BRL - Brazilian Real' => '',
-    // 'Add a new Kanboard task' => '',
-    // 'Subtask not started' => '',
-    // 'Subtask currently in progress' => '',
-    // 'Subtask completed' => '',
-    // 'Subtask added successfully.' => '',
-    // '%d subtasks added successfully.' => '',
-    // 'Enter one subtask by line.' => '',
-    // 'Predefined Contents' => '',
-    // 'Predefined contents' => '',
-    // 'Predefined Task Description' => '',
-    // 'Do you really want to remove this template? "%s"' => '',
-    // 'Add predefined task description' => '',
-    // 'Predefined Task Descriptions' => '',
-    // 'Template created successfully.' => '',
-    // 'Unable to create this template.' => '',
-    // 'Template updated successfully.' => '',
-    // 'Unable to update this template.' => '',
-    // 'Template removed successfully.' => '',
-    // 'Unable to remove this template.' => '',
-    // 'Template for the task description' => '',
-    // 'The start date is greater than the end date' => '',
-    // 'Tags must be separated by a comma' => '',
-    // 'Only the task title is required' => '',
-    // 'Creator Username' => '',
-    // 'Color Name' => '',
-    // 'Column Name' => '',
-    // 'Swimlane Name' => '',
-    // 'Time Estimated' => '',
-    // 'Time Spent' => '',
-    // 'External Link' => '',
-    // 'This feature enables the iCal feed, RSS feed and the public board view.' => '',
-    // 'Stop the timer of all subtasks when moving a task to another column' => '',
-    // 'Subtask Title' => '',
-    // 'Add a subtask and activate the timer when moving a task to another column' => '',
-    // 'days' => '',
-    // 'minutes' => '',
-    // 'seconds' => '',
-    // 'Assign automatically a color when preset start date is reached' => '',
-    // 'Move the task to another column once a predefined start date is reached' => '',
+	'number.decimals_separato
```

Comment:
The placeholder `%s` is missing here.

Reply:
Yes I found it and some other minors, should I make a new PR then??

Reply:
You could update this PR until the tests are green or make a new one. It's up to you. Thanks